templates/CMakeLists.txt.template

```diff
@@ -40,17 +40,17 @@
   # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
   <%!
-  
+
```

Comment:
Is the default order of package search that find_package is doing undesirable?
Looks like it first tries to find -module type of package and if it can't, it will proceed to look for -config type of package.

I guess the usecase that you have in mind is that there are both available on the machine, but the one of -config type is newer / more up-to-date than the other. Isn't it possible to use the  CMAKE_MODULE_PATH  to override the path it searching in or even use gRPC_PROTOBUF_PROVIDER=module (with -DPROTOBUF_ROOT_DIR=...) ?